### PR TITLE
Setting Action Cable

### LINKED ISSUE #804: Setup action cable for rails 5
Action cable is required for setting up realtime environment for synchronous editing.
build on top of #685 
part of #593

### PR DESCRIPTION
**Pull request by alaxalves**
Fixes #804 (<=== Add issue number here)

This is a replication of https://github.com/publiclab/mapknitter/pull/805 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

[//]: # (To mark checkbox write 'x' within the square brackets)

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/mapknitter-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by ViditChitkara** [comment]
This is great!! So, this would be merged in main? Also, I've added a few bug fixes and additions in #957 . If you could have a look there, then we can merge both of these at the same time.
Thanks.

**Comment by alaxalves** [comment]
> This is great!! So, this would be merged in main? Also, I've added a few bug fixes and additions in #957 . If you could have a look there, then we can merge both of these at the same time.
> Thanks.

https://github.com/publiclab/mapknitter/pull/957 Looks ok to me, can you take care of the Codeclimate offenses here and the Rubocop issues there? @ViditChitkara

**Comment by jywarren** [comment]
Hi @ViditChitkara @alaxalves can you update on the status here? Thanks so much!

**Comment by alaxalves** [comment]
> Hi @ViditChitkara @alaxalves can you update on the status here? Thanks so much!

It's good to go! I have just updated with the changes @ViditChitkara has made regarding this on https://github.com/publiclab/mapknitter/pull/957. When the build passes you can merge this!

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -12,41 +12,42 @@
 //
 
 
-//= require leaflet/dist/leaflet-src.js
+// = require leaflet/dist/leaflet-src.js
```
Comment (jywarren) [inline app/assets/javascripts/application.js]: Hi @alaxalves this is re-introducing these spaces which cause these JS assets not to be compiled in. If this is intended, could you explain why? Just seeing this was recently undone by @kaustubh-nair and I feel like we may be crossing wires communication-wise here. Thanks!!! 👍

Comment (alaxalves) [reply app/assets/javascripts/application.js]: Actually it was Rubocop's `rubocop --auto-fix` task. I will undo this. Thx for pointing it out!

Comment (alaxalves) [reply app/assets/javascripts/application.js]: @jywarren Just fixed this! Could you check it out again please?

Comment (jywarren) [reply app/assets/javascripts/application.js]: aha! Rubocop! 😄 Thank you!

**Comment by jywarren** [comment]
We'll try to get this onto stable now!

**Pull request merged**